client/src/components/pages/Stats.css

```diff
@@ -21,3 +21,18 @@
   margin-right: var(--m);
 }
 
+.seed-1 {
```

Comment:
there's also seed colors that show on TeamCard defined here: https://github.com/cory2067/gtsosu/blob/baeca928308d0f3a4ba1b6235c7c7db7de15cae5/client/src/components/modules/TeamCard.css#L48

maybe we'd wanna make these consistent at some point

Reply:
oh, maybe. I think I'd be okay with them being different though :^)